styles.css

```diff
@@ -144,6 +142,89 @@ h1 {
   width: 10.635416984558105rem;
 }
 
+#hamburger-menu {
+  background-image: url(images/Overlay.png);
+
+  /* height: 37.75rem;
+  width: 23.4375rem; */
```

Comment:
- [ ] As I pointed out above about leaving code as a comment, I'll suggest you remove this too. :+1: :wave: